### PR TITLE
Desktop version portfolio

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
- This project involved making my portfolio responsive
- I will be adding media queries to the mobile version portfolio so that it is responsive and adaptable to most screens.
- These sections of the mobile website include:
    1. The toolbar section
    2. Works section
    3. The About myself
    4. The contact section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+    margin-top: 91px;
+    margin-left: 67px;
+  }
+
+  @media (min-width: 768px) {
+    #media {
+      align-items: flex-start;
+      margin-top: -41px;
+      margin-left: -130px;
+    }
```
Comment (mgmediaweb) [inline style.css]: Kindly, I suggest you use only 1 media query for your desktop version

File: style.css [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+  width: 23.5rem;
+  height: 735px;
+  background: #fff;
+  margin-top: 131px;
+  border-bottom-left-radius: 100px;
+  margin-bottom: 80px;
+  background-image: url(./images/background.png);
+  background-repeat: no-repeat;
+  background-position-y: 100px;
+}
```
Comment (mgmediaweb) [inline style.css]: Kindly, I recommend you to use flexbox to adapt the information to the center of the container

File: style.css [hunk 3]
```diff
@@ -0,0 +1,947 @@
...
+  background-repeat: no-repeat;
+  background-position-y: 9px;
+  background-position-x: 153px;
+}
+
+#contact-me {
+  width: auto;
+  height: 56px;
+  text-align: center;
+  margin-top: 78px;
```
Comment (mgmediaweb) [inline style.css:685]: Kindly, I recommend you to use flexbox to adapt the information to the center of the container

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Kevo-hacker, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost4

## Highlights 💯 
- Good descriptive PR ✔️ 
- Professional looking readme ✔️ 
- Good applied design ✔️ 

## Required Changes ♻️

- Kindly, I suggest you remove this white space, it does not make your website look good.
- Kindly, this text is not the original font of the Figma design, I recommend you change it to improve the appearance of your page
image

- Kindly, this text is not the original font of the Figma design, I recommend you change it to improve the appearance of your page
image

Kindly, this black line does not belong to the original design, besides that your page has excessive space in the background, they suggested you remove them so that your page looks more PRO 😉 
image

Kindly, this section doesn't look good. I recommend you try a higher height and better adapt the background so that it does not overlap with the texts
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Kevo-hacker,

# STATUS: APPROVED  🎉🎉🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**